### PR TITLE
Harden xss security fix

### PR DESCRIPTION
**Pull request by klemens-st**
@picandocodigo I realised the fix from #526 works well when someone uses `excerpt_tag=script` but if someone tries `excerpt_tag="s(cript"` the fix is ineffective, because `to_html` receives this value, escapes it with `tag_escape` so in the end it becomes `script`.

This small change prevents that.

### CODE REVIEW

**Review by picandocodigo — APPROVED** [review]
I was just working on adding more stuff to this same code! I'll add the commit to your branch.

Do we need to install `svn` manually on the GitHub Action?

**Comment by picandocodigo** [comment]
Looks like the svn error was something else :shrug:

**Comment by klemens-st** [comment]
> Do we need to install svn manually on the GitHub Action?

Yes we do. I've just fixed it 😄

**Comment by picandocodigo** [comment]
Thanks! Hopefully this will be the last fix for `script` :sweat_smile: 
I'll add the readme changes to this branch and if everything looks good on your side too we can tag 0.90.3

File: tests/lcpwrapper/test-wrap.php
Comment (klemens-st) [inline tests/lcpwrapper/test-wrap.php:1]: Yeah, that's great

**Comment by klemens-st** [comment]
Looks good, I'm almost sure there's no way to squeeze in `script` now 😆

**Comment by klemens-st** [comment]
Oh, I forgot to fix the deploy action so it failed, I'll fix it now.

**Comment by klemens-st** [comment]
It should be good now

**Comment by klemens-st** [comment]
Sorry again for the mess 😢 I've now fixed the deploy workflow and deleted the 0.90.3 tag so you can make the release again

**Pull request merged**